modules/account-lib/src/coin/sol/tokenEncodeDecode.ts

```diff
@@ -0,0 +1,173 @@
+/**
+ * @prettier
+ */
+import { AccountMeta, PublicKey, TransactionInstruction } from '@solana/web3.js';
+const splToken = require('@solana/spl-token');
+const BigNumber = require('bignumber.js');
+
+/*
+  IMPORTANT:
+
+  Since token transfer encoding and decoding is in alpha (v 0.2.0-alpha.2), a version that is currently not functional due to typing bugs,
+  this file is the cherry-picked methods that are required to complete the token transfer feature. When the Solana foundation releases a stable
+  version, this file should be deleted and the spl-token methods should be used.
+ */
+
+export enum TokenInstruction {
```

Comment:
let's minimize the models/functions we're exporting.
i feel like we really only need to export `decodeTransferCheckedInstruction` and `DecodedTransferCheckedInstruction`